src/containers/BookForm.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const categories = [
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
+  'Sci-Fi',
+];
+const makeoption = categories.map((item, index) => (
+  <option key={index.toString()}>
+    {item}
+  </option>
+));
+
+const BookForm = () => (
+  <div>
```

Comment:
Kindly change `<div>` to `<form>`

Reply:
These changes are referred to the next milestone, this pull request is for milestone 2